x-pack/plugins/inference/scripts/load_esql_docs/load_esql_docs.ts

```diff
@@ -5,30 +5,30 @@
  * 2.0.
  */
 import { run } from '@kbn/dev-cli-runner';
+import { ESQLMessage, EditorError, getAstAndSyntaxErrors } from '@kbn/esql-ast';
+import { validateQuery } from '@kbn/esql-validation-autocomplete';
 import $, { load } from 'cheerio';
 import { SingleBar } from 'cli-progress';
 import FastGlob from 'fast-glob';
 import Fs from 'fs/promises';
-import { once, partition, compact } from 'lodash';
+import { compact, once, partition } from 'lodash';
 import pLimit from 'p-limit';
 import Path from 'path';
 import git, { SimpleGitProgressEvent } from 'simple-git';
 import yargs, { Argv } from 'yargs';
-import { MessageRole } from '@kbn/observability-ai-assistant-plugin/common';
-import { validateQuery } from '@kbn/esql-validation-autocomplete';
-import { EditorError, ESQLMessage, getAstAndSyntaxErrors } from '@kbn/esql-ast';
-import { connectorIdOption, elasticsearchOption, kibanaOption } from '../evaluation/cli';
-import { getServiceUrls } from '../evaluation/get_service_urls';
-import { KibanaClient } from '../evaluation/kibana_client';
-import { selectConnector } from '../evaluation/select_connector';
+import { lastValueFrom } from 'rxjs';
+import { INLINE_ESQL_QUERY_REGEX } from '../../common/tasks/nl_to_esql/constants';
+import { correctCommonEsqlMistakes } from '../../common/tasks/nl_to_esql/correct_common_esql_mistakes';
+import { connectorIdOption, elasticsearchOption, kibanaOption } from '../util/cli_options';
+import { getServiceUrls } from '../util/get_service_urls';
+import { KibanaClient } from '../util/kibana_client';
+import { selectConnector } from '../util/select_connector';
 import { extractSections } from './extract_sections';
-import { correctCommonEsqlMistakes } from '../../server/functions/query/correct_common_esql_mistakes';
-import { INLINE_ESQL_QUERY_REGEX } from '../../server/functions/query/constants';
 
 yargs(process.argv.slice(2))
   .command(
     '*',
-    'Extract ES|QL documentation for the Observability AI Assistant',
+    'Extract ES|QL documentation',
```

Comment:
No objections about me moving the ESQL doc generation script to a package in a follow-up? it's more consistent with how we're been working with scripts in AppEx, and I like the isolation of concerns (especially given we will likely be adding more extraction scripts later, e.g. for the elastic.co doc)

Reply:
no, that's a good idea!